### DISCUSSION TITLE
Issue with npm install

### DISCUSSION THREAD

**Discussion by Davidthecode** [rad-ui/ui#236] [Q&A] [answered]
First of all, hats off to those building this cool UI component library. I came across this project, loved the idea and wanted to contribute but i had issues installing the node modules when i ran  `npm install`.  I encountered an error that indicated a dependency resolution issue with conflicting versions of the Rollup package and its plugins. 
Since i couldn't figure out the issue, i ran a force install only to encounter more issues related to the `node-sass` module. I did some research and found out its a native addon that's required to be compiled during installation and a build tool is required - `node-gyp`. 
I went through the `node-gyp` docs for windows but i have been unable to make it work on my machine. I could use some assistance to resolve this issue and get it running on my PC. Thanks in advance for your help.

**Comment by Davidthecode** [comment]
@kotAPI @CCleanerShot

**Reply by kotAPI** [reply]
Hey @Davidthecode so sorry, didn't get the notification until you tagged me again. Yes, you're right, roll-up dependencies have been an ongoing issue, we thought of uninstalling them and reinitialising them again.

I'm down with health issues and won't be back online until a few days, let me find a solution and tag you back?

**Reply by Davidthecode** [reply]
Thanks for your response, @kotAPI. Sure, I'll be waiting to hear your solution!

**Reply by kotAPI** [reply]
try this out @Davidthecode https://github.com/rad-ui/ui/pull/224

**Reply by Davidthecode** [reply]
@kotAPI what is the script to start the development

**Reply by kotAPI** [reply]
@Davidthecode npm run sb

**Reply by kotAPI** [reply]
Why don't you join the discord channel?

**Reply by Davidthecode** [reply]
I have Joined already

**Comment by kotAPI** [comment, marked as answer]
This issue was being caused due to eslint parser package, which has been removed
